runtime/drivers/duckdb/duckdb.go

```diff
@@ -130,7 +130,20 @@ func (d Driver) Open(cfgMap map[string]any, shared bool, ac activity.Client, log
 	// Open the DB
 	err = c.reopenDB()
 	if err != nil {
-		return nil, err
+		if c.config.ErrorOnIncompatibleVersion || !strings.Contains(err.Error(), "created with an older, incompatible version of Rill") {
+			return nil, err
+		}
+		c.logger.Named("console").Info("removing db file created using old version")
+		tmpPath := cfg.DBFilePath + ".tmp"
```

Comment:
```suggestion
		}

		c.logger.Named("console").Info("Resetting .db file because it was created with an older, incompatible version of Rill")

		tmpPath := cfg.DBFilePath + ".tmp"
```